cluster/gce/util.sh

```diff
@@ -88,11 +88,11 @@ function set-linux-node-image() {
 function set-windows-node-image() {
   WINDOWS_NODE_IMAGE_PROJECT="windows-cloud"
   if [[ "${WINDOWS_NODE_OS_DISTRIBUTION}" == "win2019" ]]; then
-    WINDOWS_NODE_IMAGE="windows-server-2019-dc-core-for-containers-v20200908"
+    WINDOWS_NODE_IMAGE="windows-server-2019-dc-core-v20210112"
   elif [[ "${WINDOWS_NODE_OS_DISTRIBUTION}" == "win1909" ]]; then
-    WINDOWS_NODE_IMAGE="windows-server-1909-dc-core-for-containers-v20200908"
-  elif [[ "${WINDOWS_NODE_OS_DISTRIBUTION}" == "win1809" ]]; then
-    WINDOWS_NODE_IMAGE="windows-server-1809-dc-core-for-containers-v20200908"
+    WINDOWS_NODE_IMAGE="windows-server-1909-dc-core-v20210112"
+  elif [[ "${WINDOWS_NODE_OS_DISTRIBUTION}" == "win20h2" ]]; then
```

Comment:
Can we add 2004 as well even if it's unused?

Reply:
Done.